src/views/setup/store.ts

```diff
@@ -97,6 +98,83 @@ export const useSetupStore = defineStore("setup", () => {
   let toolsFolder: Ref<string> = ref("");
   let toolsResults: Ref<IEspIdfTool[]> = ref([]);
   let extensionVersion: Ref<string> = ref("");
+  let idfPathError: Ref<string> = ref("");
+  let isInstallButtonDisabled: Ref<boolean> = ref(false);
+
+  function clearIdfPathError() {
```

Comment:
Isn't this function just `setIdfPathError("");` We could remove this function I think. You also use `setIdfPathError("");` for canAccessFileResponse window event listener.